app/controllers/sessions_controller.rb

```diff
@@ -8,7 +8,7 @@ def new
 
   def create
     user = User.find_by(email: params[:email])
-    if user.authenticate(params[:password])
+    if user && user.authenticate(params[:password])
```

Comment:
On my end, it fixed the authenticate nil class error.
Please let me know if it didn't work on your end

Reply:
That worked!

Reply:
Nice work!